source/_components/octoprint.markdown

```diff
@@ -18,19 +18,38 @@ ha_iot_class: "Local Polling"
 
 To get started with the OctoPrint API, please follow the directions on their [site](http://docs.octoprint.org/en/master/api/general.html). Once OctoPrint is configured you will need to add your API key and host to your `configuration.yaml`. 
 
+Single printer:
 
 ```yaml
 octoprint:
-  host: YOUR_OCTOPRINT_HOST
-  api_key: YOUR_API_KEY
-  bed: false
-  number_of_tools: 1
+  - host: YOUR_OCTOPRINT_HOST
+    api_key: YOUR_API_KEY
+    bed: false
+    number_of_tools: 1
+```
+
+Multiple printers:
```

Comment:
Please move addition examples below the "Configuration Variables" section.

Reply:
Done.